frontend/src/components/App/Home/index.tsx

```diff
@@ -238,6 +239,10 @@ function HomeComponent(props: HomeComponentProps) {
       .sort();
   }
 
+  async function handleDynamicClusterName(cluster: string) {
+    findKubeconfigByClusterName(cluster);
+  }
```

Comment:
Why do we need to have a handleDynamicClusterName function wrapping the new one? (and in general "handle*" is not a great name unless it's for something that's just a response, e.g. `handleConfigRouteRequest`).